client/state/sharing/keyring/actions.js

```diff
@@ -48,3 +49,30 @@ export const deleteKeyringConnection = ( connection ) => ( dispatch ) => dispatc
 	type: KEYRING_CONNECTION_DELETE,
 	connection,
 } );
+
+/**
+ * Triggers a network request to delete a Keyring connection from the server-side.
+ *
+ * @param  {Object} connection         Connection to be deleted.
+ * @param  {Number} connection.ID      ID of the connection to be deleted.
+ * @param  {String} connection.label   Name of the service that was connected.
+ * @return {Function}                  Action thunk
+ */
+export function deleteStoredKeyringConnection( connection ) {
```

Comment:
Instagram connection is the only one that uses this action for now. The publicize-able sharing connections don't need this because they use a different action `deleteSiteConnection()` to remove connections. Eventbrite does not use this too. It removes the blog option `eventbrite_api_token` rather than the connected keyring when a user disconnects Eventbrite.